R/vfb_nblast.R

```diff
@@ -71,7 +71,7 @@ vfb_nblast<-function(query, n=50, target=c("FlyCircuit", "GMR-Gal4"),
     if(target=="GMR-Gal4")
       res[['id']]=gmr_vfbid(res[['id']])
     else
-      res[['id']]=vfb_tovfbids(res[['id']])
+      res[['id']]=vfb_tovfbids(res[['id']], fixed=FALSE)
```

Comment:
@Robbie1977 does fixed work at all now? see https://github.com/jefferis/vfbr/issues/19

Reply:
@Robbie1977 in #19 I change the query field. The problem with what you have done here is that the results will no longer be in the order of the input ids, which could be a big problem. `fixed=FALSE` was intended for wildcard queries which might produce different numbers of outputs from the number of supplied input ids.